tools/subgraph-example/subgraph/subgraph.yaml

```diff
@@ -0,0 +1,69 @@
+specVersion: 0.0.5
+schema:
+  file: ./schema.graphql
+dataSources:
+  - kind: ethereum
```

Comment:
Q: Could this be changed to Hedera if not in summary what would be needed to make this work for a self hosted option

Reply:
Here `ethereum` is the network type or as they also call it the `protocol`, it actually means it's an `evm compatible` network. Basically it tells all the tools how to treat that subgraph, what triggers it should look for (events, calls, blocks), how to convert Solidity types to AssemblyScript types (the language the subgraphs are written) and vice versa, how to call Solidity contracts, etc. 

It will most likely require forking and renaming it in the graph-node, graph-cli and graph-ts

Reply:
Gotcha, wish they'd name it `evm-protocol` instead :)